src/pytype.jl

```diff
@@ -280,7 +280,7 @@ type PyTypeObject
              Py_TPFLAGS_HAVE_CLASS |
              Py_TPFLAGS_HAVE_STACKLESS_EXTENSION |
              Py_TPFLAGS_HAVE_INDEX)
-        name_save = bytestring(name)
+        name_save = Base.cconvert(Ptr{UInt8}, name)
```

Comment:
I'm not sure what you're talking about.  CPython never looks at the `tp_name_save` field; this is is something I tacked on to the end of the struct so that the Julia-side object would cache a reference to the string to make sure it is not garbage collected for as long as I keep the Julia type object around (forever).